.github/workflows/dependency-review.yml

```diff
@@ -17,5 +17,5 @@ jobs:
         uses: actions/dependency-review-action@v4
         with:
           fail-on-severity: moderate
-          allow-licenses: Apache-2.0, MIT, BSD-2-Clause, BSD-3-Clause, ISC, CC0-1.0, Unlicense
+          allow-licenses: Apache-2.0, MIT, BSD-2-Clause, BSD-3-Clause, ISC, CC0-1.0, Unlicense, Null, BlueOak-1.0.0
```

Comment:
Seems like adding `Null` here is making the action fail. Maybe an alternative solution is to add [jackspeak](https://github.com/isaacs/jackspeak) to [allow-dependencies-licenses](https://github.com/actions/dependency-review-action?tab=readme-ov-file#configuration-options).

Reply:
hmm, good catch, it seemed like Null was a license ID, but looking at the license list, that's a non-existing identifier. Will update.

Reply:
Done. Thanks for pointing it out.